controllers/cart_controller.rb

```diff
@@ -24,6 +25,11 @@ def add
       current_orderable.destroy
 
     else
+      #@cart.validate_age_restriction_for_products
+     
+    
+        @product.min_age_required.present?
```

Comment:
why is this required here?

Reply:
updating code today , i was experimenting with code